packages/typespec-ts/test/modularUnit/scenarios/models/models.md

```diff
@@ -0,0 +1,96 @@
+# generate constant type model
+
+## TypeSpec
+
+```tsp
+model StreamingChatCompletionOptions {
+  stream: true;
+  messages: "aaaaa";
+  index: 123;
+}
+op read(@path id: string): {
+  @bodyRoot result: StreamingChatCompletionOptions
```

Comment:
Please also set request body for this so that we can test both request and response. thanks